src/utils/cache-helper.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+function directives(expression = '') {
```

Comment:
what is this for?
can this be covered by https://github.com/kornelski/http-cache-semantics ? (which is already used in helix-fetch)

Reply:
I did check `http-cache-semantics` before I started writing, but I got no luck. The main thing I need is the `merge` function, which determines the tightest possible cache timeouts when multiple data sources are used in one document.